edge/Makefile

```diff
@@ -28,3 +28,12 @@ integration_test:
 .PHONY: clean
 clean:
 	bash -x hack/clean.sh
+
+.PHONY: cross_build
+cross_build:
+	export GOARCH=arm
```

Comment:
So the default arch is `amd` right? What about
```
for arch in ("amd", "arm"); then
do
 export  GOARCH=$arch
 go build
done
```

Reply:
How about we keep this for loop for all other h/w platforms we support and amd we keep seperate as a basic one ??

Reply:
OK, no problem. Either way it ok to me :)

Reply:
Just for clarification :)

Reply:
See this : http://www.codershaven.com/multi-platform-makefile-for-go/

can we try to do something similar ?